Automattic/newspack-theme
# feat: add social icon for Patreon

### All Submissions:


### Changes proposed in this Pull Request:

This PR adds an icon for Patreon links to the social links menu.

Closes #1574.

### How to test the changes in this Pull Request:

1. Apply the PR. 
2. Add or edit a menu assigned to the 'Social Links Menu' location; add a link to a Patreon account (eg. https://www.patreon.com/username).
3. Confirm that the menu uses the correct icon, and not the 'link' icon (second to last): 

![image](https://user-images.githubusercontent.com/177561/142030245-2f77a034-5463-42a2-9cb4-ab3add8d7ea2.png)

### Other information:
